src/main/resources/db/migration/V2__create_item_cat_auction.sql

```diff
@@ -0,0 +1,43 @@
+BEGIN;
+CREATE TABLE IF NOT EXISTS category (
+  id UUID DEFAULT gen_random_uuid () PRIMARY KEY,
+  name text NOT NULL,
+  subcategory UUID,
+  CONSTRAINT fk_subcategory
+      FOREIGN KEY(subcategory) 
+	  REFERENCES category(id)
+);
+
+CREATE TABLE IF NOT EXISTS item (
+  id UUID DEFAULT gen_random_uuid () PRIMARY KEY,
+  name text NOT NULL,
+  start_price decimal,
+  color text,
+  size numeric,
+  description text NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS auction (
+  id UUID DEFAULT gen_random_uuid () PRIMARY KEY,
+  start_date timestamp,
+  end_date timestamp,
+  highest_bid decimal,
+  address text,
+  zip_code numeric,
+  phone text,
```

Comment:
Now I remembered that we discussed this when I created the initial ERD, whether the user will have to enter these details for every shipment, if yes, I should put it in the payment details table, if it needs to enter it just once, I should put it in the user account table. You mentioned you would check this, as it is unclear in the design files.

In which table should I move these fields then?

Reply:
I'll get back to you on that as soon as I have all the information necessary. For now, they can be removed completely, because you can add columns to existing tables through subsequent migrations if necessary and I don't believe you need to display these anywhere as of right now.

Reply:
Okaay